projects/fourier/lib/fourier/utilities/xcode.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "json"
+
+module Fourier
+  module Utilities
+    module Xcode
+      def self.current_xcode_version
+        %x{ xcode-select -p }.split(/(?<=app)/).first
+      end
+
+      def self.switch_xcode_version(xcode_path)
+        Utilities::System.system("sudo xcode-select -switch #{xcode_path}")
```

Comment:
Is it really necessary to do this? I can think of a few reasons not to:

1. Using `sudo` should be avoided if possible, and may be impossible for some users.
2. Switching Xcode version system-wide could cause issues if other tasks are running at the same time
3. In case of an error, the switch-back might not happen, leaving the user with an unexpected Xcode version.

`xcode-select` recommends using the `DEVELOPER_DIR` environment variable instead. This seems like the better approach.

Reply:
Good call, using `DEVELOPER_DIR` seems like a much better approach.

- [ ] Update to use `DEVELOPER_DIR` env variable instead of switching Xcode versions

Reply:
This is using `DEVELOPER_DIR` now 👍